### DISCUSSION TITLE
[Feature] Support manual re-ordering photos in albums

### DISCUSSION THREAD

**Discussion by spencerericfong** [immich-app/immich#2208] [Feature Request]
### The feature

It would be great to be able to manually re-order photos in albums so you can organize them in a specific order that you want. For example, by dragging and dropping individual photos within the album (like how the App-Which-Must-Not-Be-Named does it).

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by clemmrich** [comment]
For me, this would be the central feature to go from a small tool to get photos from smartphone to Lightroom to a true photo management solution, which together with some of the existing free editing tools, could replace Lightroom altogether. 

Update: no longer relavant for me since I completely erased Immich since without this feature it is absolutely useless for my use case.

**Comment by marshalleq** [comment]
Or even just, display oldest first, something the event photographers and story tellers want as a bare minimum.

**Comment by y0c4** [comment]
This is also a feature my entire family tells me is the reason why they cannot use Immich more consistently

**Comment by vicleac-git** [comment]
Make it real!

**Comment by mryoaz** [comment]
I would also love to see this feature. For me, album is more then just a collection of photo's. It's some sort of a story. And that story can be rightfully told with a custom order other then by timestamp.

**Comment by skyynet** [comment]
Photos from recent years are usually tagged with their correct dates in the meta information. For older photos, which were scanned, either the scan time or none is stored. Sometimes you don't want to show photos in the order of their time when they were taken but in another structured way.

I also would like to see a feature to sort photos in albums either by filename (photo-001.jpg, photo-002.jpg, ...) or manually to have my intended order when presenting them in a slideshow.
